### PR TITLE
tf/modules/r53/main.tf: Change LDAP DNS records

### PR DESCRIPTION
**Pull request by RyanSquared**
These are already deployed, and LDAP is working again.

### CODE REVIEW

**Review by DeviaVir — APPROVED** [review]

**Comment by KellerFuchs** [comment]
@DeviaVir @RyanSquared Why is this deployed, approved by not merged?

**Comment by DeviaVir** [comment]
There have been updates to `master` since this PR, which is why I can't merge it on top of latest merge with `master` (since all commits require approval)

@RyanSquared if you can merge latest master and push it up, we can re-approve and get it merged.

**Comment by KellerFuchs** [comment]
@DeviaVir Doesn't look like it.

**Pull request merged**
